File: Coding Practice/C++/Practice2/circle.cpp
#include<iostream>

using namespace std;

int main()
{
    const float pi = 3.14;
    float r;
    cout<<"Enter radius of a circle: ";
    cin>>r;
    cout<<"Area of the circle is: "<<pi*r*r<<endl;
    cout<<"Circumference of the circle is: "<<2*pi*r<<endl;
    return 0;
}

File: Coding Practice/Java/Core_Java/Day09/Circle.java
// Homework problem 04 :
// Write a function that takes in the radius as input and returns the circumference of a circle.

package Core_Java.Day09;
import java.util.Scanner;

public class Circle {
    static final double PI = 3.14159265359;

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter radius of a circle : ");
        double rad = sc.nextDouble();
        double res = circle(rad);
        System.out.printf("Circumference of circle with %.6f radius id : %.6f", rad, res);
    }

    public static double circle(double rad) {
        return 2 * PI * rad;
    }
}
